app/models/Customer.py

```diff
@@ -0,0 +1,37 @@
+from app import db
+from flask import current_app
+from sqlalchemy import DateTime
+from datetime import date, datetime, timedelta
+
+class Customer(db.Model):
+    __tablename__ = "customers"
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True, default=datetime.utcnow())
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+    rentals = db.relationship("Rental", backref="customers", lazy=True)
+    
+
+    def customer_info(self):
+        if self.registered_at:
+            is_registered = True
+        else:
+            is_registered = False
```

Comment:
It doesn't look like the `is_registered` variable being assigned here is being used for anything.